src/views/SignIn.vue

```diff
@@ -1,6 +1,23 @@
 <template>
   <div class="sign-in">
-      <h2>Sign In</h2>
+    <div class="container">
```

Comment:
add "container-sign-in" class, so it can look like <div class="container container-sign-in">